.github/workflows/ci.yml

```
@@ -258,6 +258,31 @@ jobs:
         DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         DOCKERHUB_REPOSITORY: ${{ secrets.IMAGE_NAME }}
 
+  # validate the swagger docs
+  swaggerci:
+    if: github.repository == 'meshery/meshery'
+    name: swagger-docs
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: "master"
+    - name: Check if swagger was modified
+      uses: dorny/paths-filter@v2
+      id: changes
+      with:
+        filters: |
+          modified:
+            - added|modified: 'handlers'
```

```suggestion
            - added|modified: 'handlers/**'
```
since handlers is a directory and we're checking if anything in it was modified.

---

I tried doing it but the workflow failed indicating it cannot find handlers.

---

oh :thinking:

---

Should I confirm it here ?

---

No no its fine